src/Symfony/Component/Workflow/Tests/WorkflowTest.php

```diff
@@ -286,7 +286,7 @@ public function testApplyWithNotEnabledTransition()
 
             $this->fail('Should throw an exception');
         } catch (NotEnabledTransitionException $e) {
-            $this->assertSame('Transition "t2" is not enabled for workflow "unnamed".', $e->getMessage());
+            $this->assertSame('Can not apply transition to "t2" for workflow "unnamed".', $e->getMessage());
```

Comment:
IMO the change to this test shows that the change is not correct. t2 is a transition, right?

Reply:
My comment was about the assertion per se but what we expect now does not seem logical. Why do we expect a message where the name of the transaition is used in a way that makes it sound that we try to reach a place named the same as the transition?